back/engines/commercial/verification/app/serializers/verification/web_api/v1/verification_method_serializer.rb

```diff
@@ -5,8 +5,8 @@ class WebApi::V1::VerificationMethodSerializer < ::WebApi::V1::BaseSerializer
     VerificationService.new.all_methods.each do |method|
       next unless method.respond_to?(:exposed_config_parameters)
```

Comment:
not sure to follow everything here. Why don't we just skip the method if it does not match the record? something like `next unless method == record`.

Reply:
This issue with the previous code, which did that, is that if multiple methods define the config_parameters with the same name (which is the case here), the method that happens to be the last in the methods array will override the `attribute` definition.

By checking that `method == record`, the attribute wasn't included for the first method.

Reply:
Ah, so the serializer is serializing all the methods together, not just one?

Reply:
Correct